vitrina/projects/templates/vitrina/projects/datasets.html

```diff
@@ -0,0 +1,85 @@
+{% extends "base.html" %}
+{% load static %}
+{% load i18n parler_tags %}
+
+{% block pageTitle %} | {% translate "Duomenų rinkinių sąrašas" %}{% endblock %}
+{% block pageOgTitle %} | {% translate "Duomenų rinkinių sąrašas" %}{% endblock %}
+
+{% block current_title %}
+{% translate "Duomenų rinkinių sąrašas" %}
+{% endblock %}
+
+{% block content %}
+
+    {% include 'vitrina/projects/tabs.html' %}
+
+    {% if can_update_project %}
+        <div class="buttons is-right">
+            <a id="add-member-btn" class="button is-primary m-t-xs m-b-lg"
+               href="{% url 'project-dataset-add' pk=project.pk %}">
+                {% translate "Pridėti duomenų rinkinį" %}
+            </a>
+        </div>
+    {% endif %}
+    <table class="table is-fullwidth">
+        <tbody>
+            <tr>
+                <th></th>
+                {% if can_update_project %}
+                    <th></th>
+                {% endif %}
+            </tr>
+            {% for dataset in page_obj %}
+                <tr>
+                {% get_current_language as LANGUAGE_CODE %}
+                <td>
+                <article class="media">
+                    <div class="media-content">
+                        <div class="content">
+                            <div>
+                                <strong>
+                                    <a href="{{ i.object.get_absolute_url }}"class="dataset-list-title">
+                                        {% objectlanguage dataset LANGUAGE_CODE %}
+                                            {{ dataset.title }}
+                                        {% endobjectlanguage %}
+                                    </a>
+                                </strong>
+                            </div>
+                            {% if dataset.organization_title %}
```

Comment:
There is not such thing as `organization_title`. Probably you mean `organization.title`? Also if query takes data from other related models, don't forget to add select_related:

https://docs.djangoproject.com/en/3.2/ref/models/querysets/#select-related